### ISSUE TITLE
Some member variables of ares_addrinfo not set when using ares_getaddrinfo.

### ISSUE THREAD

**Issue by kedixa** [c-ares/c-ares#247]
Suppose we get `struct ares_addrinfo *addr` when `ares_getaddrinfo` callback, the following variables are not set correct in addr:
```
addr->ai_socktype
addr->ai_addrlen
(struct sockaddr_in*)(addr->ai_addr)->sin_family
(struct sockaddr_in*)(addr->ai_addr)->sin_port
// and some others in sockaddr_in6
```
these variables are left zero which will make users feel confused.

**Comment by bradh352** [comment]
should be resolved as #257

**Comment by yangboyd** [comment]
currently ai_socktype is not set.
